src/components/editor/ArticleController.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class ArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleCategorySubmit = this.handleCategorySubmit.bind(this);
```

Comment:
You don't need to bind to this at all unless you plan on using some member variables stored on the `this` object.